crates/cpp/src/lib.rs

```diff
@@ -2095,48 +2147,6 @@ impl<'a, 'b> FunctionBindgen<'a, 'b> {
             operands[0]
         );
     }
-
-    fn has_resources2(&self, ty: &Type) -> bool {
-        match ty {
```

Comment:
@TartanLlama I recall that we chatted about requiring call by value for arguments to imported functions whenever there is a resource inside. Is this included in this MR or a separate one, because it feels like this code might become handy there.